lua/nvim-tree/explorer/init.lua

```diff
@@ -60,12 +60,15 @@ function Explorer:new(args)
   self.open         = true
   self.opts         = config
 
-  self.sorters      = Sorter({ explorer = self })
-  self.renderer     = Renderer({ explorer = self })
-  self.filters      = Filters({ explorer = self })
-  self.live_filter  = LiveFilter({ explorer = self })
-  self.marks        = Marks({ explorer = self })
-  self.clipboard    = Clipboard({ explorer = self })
+
+  self.sorters     = Sorter({ explorer = self })
+  self.renderer    = Renderer({ explorer = self })
+  self.filters     = Filters({ explorer = self })
+  self.live_filter = LiveFilter({ explorer = self })
+  self.marks       = Marks({ explorer = self })
+  self.clipboard   = Clipboard({ explorer = self })
+
+  self.current_tab = vim.api.nvim_get_current_tabpage()
```

Comment:
`self.current_tab` is good. This meets the multi-instance reqirements as the explorer is now aware of its own tab and will be very useful for the remainder of the multi-instance work.